src/server/graphql/mutations/__tests__/stripeSucceedPayment.test.js

```diff
@@ -0,0 +1,43 @@
+import DynamicSerializer from 'dynamic-serializer';
+import MockDB from 'server/__tests__/setup/MockDB';
+import fetchAndSerialize from 'server/__tests__/utils/fetchAndSerialize';
+import stripe from 'server/billing/stripe';
+import stripeWebhookHandler from 'server/billing/stripeWebhookHandler';
+import getRethink from 'server/database/rethinkDriver';
+import invoicePaymentSucceededEvent from 'server/graphql/mutations/__tests__/mockStripeEvents/invoicePaymentSucceededEvent';
+import shortid from 'shortid';
+import {PRO} from 'universal/utils/constants';
+import MockRes from 'server/__mocks__/MockRes';
+import MockReq from 'server/__mocks__/MockReq';
+
+console.error = jest.fn();
+
+describe('stripeSucceedPayment', () => {
+  test('handles invoice.payment_succeeded', async () => {
```

Comment:
I just remembered - when passing functions that return promises to jest's `test`/`it`, it's best practice to use `expect.assertions(<numAssertions>)`, because a resolved promise may otherwise inadvertently "pass" the test.

https://facebook.github.io/jest/docs/en/asynchronous.html#promises

Reply:
notice that the test is `async`. desugar that & you got yourself a promise, unless i misunderstand the concern?

Reply:
You're correct about the desugaring. I was making the point that Jest will happily pass our test even if only 1 of the assertions is called (and passes). Adding the `expect.assertions()` call tells Jest to ensure that all of our assertions get called (and pass).

Reply:
ahhh, if i understand correctly, that would only be worthwhile if i'm awaiting the `expect` or i'm not doing anything with the result of what comes out of that async function. since all of our tests verify the result of what comes out of the async function, we shouldn't need to check the assertion count. If I'm still missing the picture try to dumb it down for me in an example :-)